back/engines/commercial/multi_tenancy/db/seeds/users.rb

```diff
@@ -51,36 +51,23 @@ def run
       private
 
       def run_for_empty_localhost
-        random_user = anonymizer.anonymized_attributes(AppConfiguration.instance.settings('core', 'locales'))
-        User.create!(random_user.merge({ **admin_1_attrs, id: 'e0d698fc-5969-439f-9fe6-e74fe82b567a' }))
+        User.create!(build_attrs.merge({ **ADMIN_1_ATTRS, id: 'e0d698fc-5969-439f-9fe6-e74fe82b567a' }))
       end
 
       def run_for_localhost
-        locales = AppConfiguration.instance.settings('core', 'locales')
-        User.create! anonymizer.anonymized_attributes(locales).merge(admin_1_attrs)
-        User.create! anonymizer.anonymized_attributes(locales).merge(admin_2_attrs)
-        User.create! anonymizer.anonymized_attributes(locales).merge(moderator_attrs)
-        User.create! anonymizer.anonymized_attributes(locales).merge(user_attrs)
+        [ADMIN_1_ATTRS, ADMIN_2_ATTRS, MODERATOR_ATTRS, USER_ATTRS].each do |user_attrs|
+          UserService.update_in_tenant_template!(User.new, build_attrs.merge(user_attrs))
+        end
 
         runner.num_users.times do
-          User.create! anonymizer.anonymized_attributes(locales).merge({ password: 'democracy2.0' })
+          user_attrs = build_attrs.merge({ password: 'democracy2.0' })
+          UserService.update_in_tenant_template!(User.new, user_attrs)
```

Comment:
Was a bit confused as to why this was doing an update. Might be clearer if you took `User.new` out of the method call